ios/FluentUI/Notification/NotificationTokens.swift

```diff
@@ -57,22 +57,22 @@ import UIKit
 /// Design token set for the `Notification` control.
 open class NotificationTokens: ControlTokens {
     /// Defines the style of the notification.
-    public internal(set) var style: MSFNotificationStyle = .primaryToast
+    public internal(set) var style: MSFNotificationStyle = .accentToast
```

Comment:
We may want to keep the name how it was, and then change it with the Fluent 2 Colors branch merge, rather than breaking the Notification style api.